include/arch/arm/aarch64/cpu.h

```diff
@@ -66,8 +66,44 @@
 #define SPSR_EL3_h		BIT(0)
 #define SPSR_EL3_TO_EL1		(0x2 << 1)
 
-#define __ISB()			__asm__ volatile ("isb sy" : : : "memory")
-#define __DMB()			__asm__ volatile ("dmb sy" : : : "memory")
+/* System register interface to GICv3 */
+#define ICC_IGRPEN1_EL1		S3_0_C12_C12_7
+#define ICC_SGI1R		S3_0_C12_C11_5
+#define ICC_SRE_EL1		S3_0_C12_C12_5
+#define ICC_SRE_EL2		S3_4_C12_C9_5
+#define ICC_SRE_EL3		S3_6_C12_C12_5
+#define ICC_CTLR_EL1		S3_0_C12_C12_4
+#define ICC_CTLR_EL3		S3_6_C12_C12_4
+#define ICC_PMR_EL1		S3_0_C4_C6_0
+#define ICC_RPR_EL1		S3_0_C12_C11_3
+#define ICC_IGRPEN1_EL3		S3_6_c12_c12_7
+#define ICC_IGRPEN0_EL1		S3_0_c12_c12_6
+#define ICC_HPPIR0_EL1		S3_0_c12_c8_2
+#define ICC_HPPIR1_EL1		S3_0_c12_c12_2
+#define ICC_IAR0_EL1		S3_0_c12_c8_0
+#define ICC_IAR1_EL1		S3_0_c12_c12_0
+#define ICC_EOIR0_EL1		S3_0_c12_c8_1
+#define ICC_EOIR1_EL1		S3_0_c12_c12_1
+#define ICC_SGI0R_EL1		S3_0_c12_c11_7
```

Comment:
Minor nits: I know this was copied from somewhere else, but it would still be nice to choose either all uppercase or lowercase for the reg names; for instance, we currently have:
```
#define ICC_IGRPEN1_EL1		S3_0_C12_C12_7
#define ICC_IGRPEN1_EL3		S3_6_c12_c12_7
```

Reply:
ACK
Thanks, I missed (It took awhile for me to figure out which one is lower case even after you pointed out :)).  Actually I was using lower case when I was using them directly in assembly and for some reason upper case defines were not being recognized.

Reply:
done.